addons/nodot/characters/Fly3D.gd

```diff
@@ -0,0 +1,143 @@
+## This node is a mechanic for fly player (experimental)
+class_name Fly3D extends Node3D
+
+## This signal call in "_physics_process()" of player: $(Fly3D).emit_signal("config_fly", self, $(model of player))
+signal config_fly(Parent, model)
```

Comment:
Would prefer to use the get_parent setup like the other movement nodes. Sending a signal means we would have to modify the character controller and break modularity.

Also lower case the parent argument